pages/emailSender.tsx

```diff
@@ -435,120 +357,29 @@ const EmailSender: NextPage = () => {
               ))}
             </SectionContainer>
           </FormControl>
-          <StyledTableContainer>
-            <TableContainer component={Paper}>
-              <StyledTable aria-label="uploaded csv table">
-                <TableHead>
-                  {headerKeys.map((key) => (
-                    <TableCell key={nanoid()}>
-                      <StyledBoldTypograhy variant="body1">
-                        {key}
-                      </StyledBoldTypograhy>
-                    </TableCell>
-                  ))}
-                </TableHead>
-                <TableBody>
-                  {csvRowsArray.map((item) => (
-                    <StyledTableRow key={nanoid()}>
-                      {Object.values(item).map((val) => (
-                        <TableCell key={nanoid()} align="left">
-                          {val}
-                        </TableCell>
-                      ))}
-                    </StyledTableRow>
-                  ))}
-                </TableBody>
-              </StyledTable>
-            </TableContainer>
-          </StyledTableContainer>
-          <SectionContainer>
-            <StyledSubHeader variant="h5">
-              4) Verify final messages
-            </StyledSubHeader>
-            <StyledButton
-              color="info"
-              variant="contained"
-              onClick={createMessages}
-              disabled={csvRowsArray.length === 0}
-              width="medium"
-            >
-              Print final messages
-            </StyledButton>
-          </SectionContainer>
+          <CSVTable
```

Comment:
I think we discussed also moving the CSVTable into the ImportCSVSection so let's do that! You can pass in  csvRowsArray={csvRowsArray} to ImportCSVSection and then calculate the header keys inside the component

Reply:
that might not be the best since the CSV table is displayed outside of the form area, and moving it would make it display within the form. What do u think boss?

Reply:
gotcha! good point we can keep it in the main file then!!